pkg/lib/config/configsource/config.go

```diff
@@ -22,8 +22,8 @@ type Config struct {
 	KubeNamespace string `envconfig:"KUBE_NAMESPACE"`
 	// KubeAppHostMapName indicates the name of app host mapping ConfigMap
 	KubeAppHostMapName string `envconfig:"KUBE_APP_HOST_MAP_NAME" default:"app-hosts"`
-	// KubeAppConfigPattern sets the RegEx that matches the app ID in ConfigMap/Secret name
-	KubeAppConfigPattern string `envconfig:"KUBE_APP_CONFIG_PATTERN" default:"^app-data-(.+)$"`
+	// KubeAppConfigPrefix sets the name prefix of app ConfigMap/Secret name
+	KubeAppConfigPrefix string `envconfig:"KUBE_APP_CONFIG_PREFIX" default:"app-data-"`
```

Comment:
Should we use labels instead to identity our ConfigMap and Secret?

Reply:
Sure, we can use labels. So:
- A label to indicate the host mapping ConfigMap, e.g. `https://authgear.com/kubernetes/host-mapping: true`
- A label to indicate the app config ConfigMap & Secret, e.g. `https://authgear.com/kubernetes/config-app-id: 9C390E3B-9030-48B0-A350-57F51572145F`

Both label names are constants and cannot be configured. Does that sounds good to you?